tests/integration/component.go

```diff
@@ -712,4 +712,67 @@ func componentTests(args ...string) {
 			Expect(stdError).To(ContainSubstring("quantities must match the regular expression"))
 		})
 	})
+
+	Context("Creating component using symlink", func() {
+		var symLinkPath string
+
+		JustBeforeEach(func() {
+			if runtime.GOOS == "windows" {
+				Skip("skipping on windows because of admin issue in go library")
```

Comment:
@mik-dass Can you please update the message to -  ```Skipping test because symlink creation on platform like Windows, go library need elevated privileges to access the system file.```